test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -995,6 +998,20 @@ func testNamespaceDelete(c *testContext) {
 	}
 }
 
+func testNamespaceCreate(c *testContext) {
```

Comment:
rather than adding a custom `testNamespaceCreate`, suggest broadening the existing special-case for namespaces in `expect`:

```diff
 func (h *holder) expect(gvr schema.GroupVersionResource, gvk, optionsGVK schema.GroupVersionKind, operation v1beta1.Operation, name, namespace string, object, oldObject, options bool) {
        // Special-case namespaces, since the object name shows up in request attributes for update/delete requests
-       if len(namespace) == 0 && gvk.Group == "" && gvk.Version == "v1" && gvk.Kind == "Namespace" && operation != v1beta1.Create {
+       if len(namespace) == 0 && gvk.Group == "" && gvk.Version == "v1" && gvk.Kind == "Namespace" {
                namespace = name
        }
```

Reply:
Changed [here](https://github.com/kubernetes/kubernetes/commit/3cb510e33eecbdc37aad14f121396ccfbf5268cb#diff-706a54bca254b383e7104ed375dd6f54).